Makefile

```diff
@@ -69,8 +69,13 @@ install-build-tools:
 	./scripts/build/fetchdeps github.com/gojuno/minimock/cmd/minimock 890c67cef23dd06d694294d4f7b1026ed7bac8e6
 	./scripts/build/fetchdeps github.com/gogo/protobuf/protoc-gen-gogoslick v1.2.1
 
+
+.PHONY: install-go-lib
+install-go-lib:
+	go get github.com/insolar/x-crypto
```

Comment:
What is it? Use `dep`.

Reply:
dep uses vendor. We need to install it worldwide